integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java

```diff
@@ -263,36 +272,28 @@ public boolean verifyQueryMetalake(String name) {
         return false;
       }
     } catch (Exception e) {
+      LOG.error(e.getMessage(), e);
       return false;
     }
   }
 
-  // ** comment for MetalakePageTest.java testLinkToCatalogsPage()
-  //  public boolean verifyLinkToCatalogsPage(String name) {
-  //    int sleepTimeMillis = 1_000;
-  //    try {
-  //      Thread.sleep(sleepTimeMillis);
-  //      String xpath = "//a[@data-refer='metalake-name-link']";
-  //      WebElement nameLink = driver.findElement(By.xpath(xpath));
-  //
-  //      Wait<WebDriver> wait =
-  //          new FluentWait<>(driver)
-  //              .withTimeout(Duration.ofSeconds(10))
-  //              .pollingEvery(Duration.ofMillis(300))
-  //              .ignoring(ElementNotInteractableException.class);
-  //      wait.until(d -> nameLink.isDisplayed());
-  //
-  //      String url = driver.getCurrentUrl();
-  //      boolean isUrl = url.contains("/ui/metalakes?metalake=" + name);
-  //
-  //      return nameLink.isDisplayed() && isUrl;
-  //    } catch (Exception e) {
-  //      return false;
-  //    } finally {
-  //      // Back to homepage
-  //      String xpath = "//*[@data-refer='back-home-btn']";
-  //      WebElement backHomeBtn = driver.findElement(By.xpath(xpath));
-  //      backHomeBtn.click();
-  //    }
-  //  }
+  public boolean verifyLinkToCatalogsPage(String name) {
+    try {
+      String xpath = "//a[@data-refer='metalake-name-link']";
```

Comment:
Can we define a link element variable in class member?

Reply:
Done